apps/parse/lib/parse/alerts.ex

```diff
@@ -112,6 +115,31 @@ defmodule Parse.Alerts do
     do_translated_text(rest, opts)
   end
 
+  defp translated_image(localizations, opts) do
+    opts =
+      opts
+      |> Map.new()
+      |> Map.put_new(:default, "")
+
+    do_localized_image(localizations, opts)
+  end
+
+  defp do_localized_image([], %{default: default}) do
+    default
+  end
+
+  defp do_localized_image(nil, %{default: default}) do
+    default
+  end
+
+  defp do_localized_image(%{"localized_image" => [%{"url" => url}]}, _) do
```

Comment:
That is the behavior I wanted, my thinking was that the language field is optional, so if there's an english one we'll take it but if there's only one image we'll use it so we still get something even if the language field isn't filled out. The odds of the field being empty are pretty low at any rate since they come straight from T-Alerts which seems to fill it out automatically.

Reply:
I don't think that's the behavior that's implemented though: I think this will take the first image, regardless of the language.

Reply:
I thought this would only pattern match on lists with item in them? Eg:
![Screenshot 2023-09-14 at 12 24 50 PM](https://github.com/mbta/api/assets/17047573/90c7c295-ea04-41c1-a1a5-4f0604b7bf9e)